src/Microsoft.TemplateEngine.Cli/Installer.cs

```diff
@@ -237,5 +265,21 @@ private void InstallLocalPackages(IReadOnlyList<string> packageNames)
                 }
             }
         }
+
+
+        internal virtual bool ExecuteProcess(string command, params string[] args)
```

Comment:
There's a helper for executing stuff at the command line that we should use instead. It handles the redirection of the output streams and all that.